cargo/spec/dependabot/cargo/file_fetcher_spec.rb

```diff
@@ -678,7 +678,7 @@
 
       stub_request(:get, url + "excluded/Cargo.toml?ref=sha")
         .with(headers: { "Authorization" => "token token" })
-        .to_return(status: 200, body: member_fixture, headers: json_header)
+        .to_return(status: 200, body: excluded_fixture, headers: json_header)
```

Comment:
Small typo (I assume?) I found when looking at https://github.com/dependabot/dependabot-core/pull/7649 for reference